### DISCUSSION TITLE
Backgroud Opacity Not Working?

### DISCUSSION THREAD

**Discussion by aktifplay** [tailwindlabs/tailwindcss#4420] [Help] [answered]
Hi How To Make The Background Opacity Working? or How To Convert All Colors To RGBA ?

**Comment by olets** [comment]
> How To Make The Background Opacity Working? 

Are you following the method in the documentation? Don't forget `backdrop-filter` https://tailwindcss.com/docs/backdrop-opacity

```html
class="backdrop-filter backdrop-opacity-…"
```

> or How To Convert All Colors To RGBA ?

You can define your own colors  https://tailwindcss.com/docs/customizing-colors#extending-the-defaults

**Comment by simonswiss** [comment, marked as answer]
Not sure if you've done that, but you must always pair a `bg-opacity-{color}` utility with a `bg-{color}` for the opacity to have any effect.

This is due to the way Tailwind uses CSS variables to compose a color + its opacity together.

This will likely be greatly simplified in Tailwind v2.2 + JIT with the new [Opacity modifier syntax](https://github.com/tailwindlabs/tailwindcss/pull/4348).

I've made a video on that feature if interested:

https://www.youtube.com/watch?v=hoFwUe97Gdc

**Comment by aktifplay** [comment]
thanks alot @simonswiss  @olets  its working now! Its only have problem with cache. thanks alot for ur great support

**Reply by simonswiss** [reply]
Glad we were able to help! 👍
